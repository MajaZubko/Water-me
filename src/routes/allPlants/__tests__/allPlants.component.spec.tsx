// import React from 'react';
// import { AllPlants } from '../allPlants.component';
// import { makeContextRenderer } from '../../../shared/utils/testUtils';

describe('AllPlants: Component', () => {
  it('should run a test', () => {
    expect(1).toEqual(1);
  });
});

// describe('AllPlants: Component', () => {
//   const component = () => <AllPlants />;
//   const render = makeContextRenderer(component);
//
//   it('should render correctly', () => {
//     const { container } = render();
//     expect(container.firstChild).toMatchSnapshot();
//   });
// });
