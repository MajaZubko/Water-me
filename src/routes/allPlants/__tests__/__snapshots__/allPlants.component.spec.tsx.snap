// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllPlants: Component should render correctly 1`] = `
.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c5.c5 {
  margin-top: 20px;
}

.c6 {
  list-style-type: none;
  margin-top: 16px;
  margin-right: 20px;
  padding: 0;
}

.c6 li {
  display: grid;
  grid-template-columns: repeat(4,150px) repeat(3,40px);
  text-align: start;
  padding-bottom: 4px;
}

.c7 {
  text-transform: uppercase;
  font-weight: bold;
  padding-bottom: 8px;
}

.c1 {
  font-family: Trebuchet MS,sans-serif;
  font-weight: bold;
  color: #000000;
}

.c2 {
  margin-bottom: 16px;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #009988;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
}

<div
  class="c0"
>
  <h1
    class="c1"
  >
    All plants
  </h1>
  <a
    class="c2"
    href="/"
  >
    Watering calendar
  </a>
  <div
    class="c3"
  >
    <div
      class="c4"
    />
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text c5"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Add plants
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <ul
      class="c6"
    >
      <li>
        <div
          class="c7"
        >
          Plant name
        </div>
        <div
          class="c7"
        >
          Location
        </div>
        <div
          class="c7"
        >
          Water needs
        </div>
        <div
          class="c7"
        >
          Last watered
        </div>
      </li>
    </ul>
  </div>
</div>
`;
